packages/eslint-plugin-pf-codemods/test/rules/v5/react-icons-remove-props.js

```diff
@@ -0,0 +1,34 @@
+const ruleTester = require("../../ruletester");
+const rule = require("../../../lib/rules/v5/react-icons-remove-props");
+
+ruleTester.run("react-icons-remove-props", rule, {
+  valid: [
+    {
+      code: `import { FrogIcon } from '@patternfly/react-icons'; <FrogIcon  />`,
+    },
+    {
+      // No @patternfly/react-core import
+      code: `<FrogIcon  />`,
```

Comment:
```suggestion
      // No @patternfly/react-icons import
      code: `<FrogIcon size color noVerticalAlign  />`,
```

Just to test that these props won't throw an error for a non PF icons import